macOS/LocalPackages/SubscriptionUI/Sources/SubscriptionUI/Preferences/PreferencesSidebarSubscriptionState.swift

```diff
@@ -29,19 +29,25 @@ public struct PreferencesSidebarSubscriptionState: Equatable {
 
     public let personalInformationRemovalStatus: StatusIndicator
     public let identityTheftRestorationStatus: StatusIndicator
+    public let paidDuckAIRestorationStatus: StatusIndicator
+    public let isPaidDuckAIEnabled: Bool
```

Comment:
Don't you agree we should move away from using `DuckAI` instead of `AIChat` in the code for symbol names?